pkg/network/multus/BUILD.bazel

```diff
@@ -18,7 +18,10 @@ go_library(
 
 go_test(
     name = "go_default_test",
-    srcs = ["nad_test.go"],
+    srcs = [
```

Comment:
this commit can be squashed please to the previous one imho
they also have the same commit msg (even though it does some other things so it should be little different unless squashed), and this commit moves the pool + the tests as required by the previous one

Reply:
Could you please elaborate what are the advantages of doing so?

I've tried to reduce the mental burden to a minimum while doing these moves.

Reply:
the commit message here if i am not wrong is the same of the previous one, and doesnt reflect what it does

once i reviewed them first time one by one, while reviewing the previous commit, i felt the changes in this commit were missing, but as they do present at the end, it is fine to have them in separate commits, both ways have pros and cons

lets just change the commit message to reflect what this commit does and then it is good enough, wdyt?

Reply:
> the commit message here if i am not wrong is the same of the previous one, and doesnt reflect what it does

The commit messages are similar, but not identical.